docs/sdks/server-sdks/ruby.md

```diff
@@ -84,71 +138,152 @@ The SDK will invoke the `log_assignment` function with an `assignment` object th
 More details about logging and examples (with Segment, Rudderstack, mParticle, and Snowplow) can be found in the [event logging](/sdks/event-logging/) page.
 :::
 
-## 3. Assign variations
+## 3. Running the SDK
 
-Assigning users to flags or experiments with a single `get_string_assignment` function:
+How is this SDK, hosted on your servers, actually getting the relevant information from Eppo?
 
-```ruby
-require 'eppo_client'
+### A. Loading Configuration
 
-client = EppoClient::Client.instance
-variation = client.get_string_assignment(
-  '<SUBJECT-KEY>',
-  '<FLAG-KEY>',
-  {
-    # Optional map of subject metadata for targeting.
-  }
-)
-```
+At initialization, the SDK polls Eppo's API to retrieve the most recent experiment configuration. The SDK stores that configuration in memory. This is why assignments are effectively instant, as you can see yourself by profiling the code above.
```

Comment:
technically, it polls our CDN, hydrated from our API. I'ts probably worth calling out that distinction.